### PR TITLE
fix: data quality alcoholic beverages category without alcohol value

### PR DESCRIPTION
**Pull request by benbenben2**
**Important**: changes are applied **only** after editing the products that are wrongly reported by the data quality issue (i.e. product that are listed with this data quality issue will be removed only after editing them again).

### What
Replaced ```$product_ref->{alcohol_value}``` by ```$product_ref->{nutriments}{alcohol_value}``` in the file in _lib/ProductOpener/DataQualityFood.pm_

### Why
There are as many "en:alcoholic-beverages-category-without-alcohol-value" data quality issues (https://hr.openfoodfacts.org/data-quality/en:alcoholic-beverages-category-without-alcohol-value) as there are "en:alcoholic-beverages" products (https://hr.openfoodfacts.org/category/en:alcoholic-beverages). But many products have the alcohol value.

### Screenshot
Screenshot of the bug (urls differ):
image

Screenshot before (left) and after (right) the devs:
image

### CODE REVIEW

**Comment by benbenben2** [comment]
I will need help for this "--assert-tidy" error.

File: lib/ProductOpener/DataQualityFood.pm [hunk 1]
```diff
@@ -1171,7 +1171,7 @@ sub check_categories ($product_ref) {
 
 	# Check alcohol content
 	if (has_tag($product_ref, "categories", "en:alcoholic-beverages")) {
-		if (!(defined $product_ref->{alcohol_value}) || $product_ref->{alcohol_value} == 0) {
+		if (!(defined $product_ref->{nutriments}->{alcohol_value}) || $product_ref->{nutriments}->{alcohol_value} == 0) {
```
Comment (stephanegigandet) [inline lib/ProductOpener/DataQualityFood.pm:1174]: We don't need the 2nd -> (I didn't know it worked in fact :) )

```suggestion
		if (!(defined $product_ref->{nutriments}{alcohol_value}) || $product_ref->{nutriments}->{alcohol_value} == 0) {
```

File: lib/ProductOpener/DataQualityFood.pm [hunk 2]
```diff
@@ -1180,8 +1180,8 @@ sub check_categories ($product_ref) {
 		}
 	}
 
-	if (    defined $product_ref->{alcohol_value}
-		and $product_ref->{alcohol_value} > 0
+	if (    defined $product_ref->{nutriments}->{alcohol_value}
```
Comment (stephanegigandet) [inline lib/ProductOpener/DataQualityFood.pm:1183]:
```suggestion
	if (    defined $product_ref->{nutriments}{alcohol_value}
```

**Comment by stephanegigandet** [comment]
> I will need help for this "--assert-tidy" error.

You need to run "make lint", the linter will change a bit the spaces etc.

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

File: lib/ProductOpener/DataQualityFood.pm (hunk 1 above)
Comment (benbenben2) [reply lib/ProductOpener/DataQualityFood.pm]: see last commit

File: lib/ProductOpener/DataQualityFood.pm (hunk 2 above)
Comment (benbenben2) [reply lib/ProductOpener/DataQualityFood.pm]: see last commit

Comment (benbenben2) [reply lib/ProductOpener/DataQualityFood.pm]: see last commit

File: lib/ProductOpener/DataQualityFood.pm [hunk 3]
```diff
@@ -1180,8 +1180,8 @@ sub check_categories ($product_ref) {
...
+		and $product_ref->{nutriments}->{alcohol_value} > 0
```
[the remark this answers is not in the document]
Comment (benbenben2) [reply lib/ProductOpener/DataQualityFood.pm]: see last commit

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**